src/browser/modules/Stream/SysInfoFrame/helpers.tsx

```diff
@@ -270,3 +272,43 @@ export const responseHandler = (setState: (newState: any) => void) =>
         : null
     })
   }
+
+export const clusterResponseHandler = (setState: any) =>
+  function (res: any) {
+    if (!res.success) {
+      setState({ error: 'No causal cluster results', success: false })
+      return
+    }
+    const mappedResult = mapSysInfoRecords(res.result.records)
+    const mappedTableComponents = mappedResult.map((ccRecord: any) => {
+      const httpUrlForMember = ccRecord.addresses.filter((address: any) => {
+        return (
+          !address.includes(window.location.href) &&
+          (window.location.protocol.startsWith('file:')
+            ? address.startsWith('http://')
+            : address.startsWith(window.location.protocol))
```

Comment:
No need to change this since it was there before, but it's quite hard to understand what we're trying to do here.

Reply:
yeah for sure. I didn't want to change it because it's been working for a long time before I removed it